etc/packaging/appimages/package_release_module.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
Copy/pasted [`/etc/package/appimages/package_release_carto.sh`](https://github.com/viamrobotics/viam-cartographer/blob/e543ccc47d7ed10094a26d0a4ea5f88dbfeb1e1c/etc/packaging/appimages/package_release_carto.sh) and slightly modified:

```bash
diff etc/packaging/appimages/package_release_carto.sh etc/packaging/appimages/package_re
lease_module.sh
10,11c10,11
<       BUILD_CHANNEL=stable appimage-builder --recipe carto_grpc_server-`uname -m`.yml
<       BUILD_CHANNEL=$CUR_TAG appimage-builder --recipe carto_grpc_server-`uname -m`.yml
---
>       BUILD_CHANNEL=stable appimage-builder --recipe cartographer-module-`uname -m`.yml
>       BUILD_CHANNEL=$CUR_TAG appimage-builder --recipe cartographer-module-`uname -m`.yml
```